### ISSUE TITLE
Sort order for contact list view not specified

### ISSUE THREAD

**Issue by david-venhoff** [digitalfabrik/integreat-cms#3252]
### Describe the Bug

Currently, the contacts in the contact list view are sorted according to their name. This is problematic for two reasons:
* Contacts for the same poi might be very far apart
* Since the name is allowed to be blank, there will be many contacts without a name. For those contacts, the sort ordering is not really specified then

### Expected Behavior

I am not sure what the best solution is. Maybe first sort by poi, then by name? This would make sure that contacts that belong to the same poi are listed together

**Comment by JoeyStk** [comment]
@osmers / @nikolahoff Does one of you have a preference on how the contacts should be ordered? For example should contacts belonging to the same POI be grouped - or do you have a different preference for sorting? :)

**Comment by osmers** [comment]
I think grouping by POI makes sense and then order the POIs by alphabetic order?

**Comment by nikolahoff** [comment]
Yes grouping by POIs is a good idea, and within the POI the contact names are ordered alphabetically right?
